mikupad.html

```diff
@@ -150,6 +157,30 @@
 #an-area {
 	min-height:3.5em;
 }
+.wi-textarea {
+	min-height: 4em;
+}
+
+#advancedContextPlaceholders {
+	background: var(--color-base-30);
+	margin: 1em auto;
+	padding: .25em;
+	border-radius: 2px;
+	width: fit-content;
+}
+#advancedContextPlaceholders th,

```

Comment:
I think the border looks better with the current theme colors:
![image](https://github.com/lmg-anon/mikupad/assets/139719567/0b0af4e0-654a-4a4d-9b5f-a79bf909647d)
```suggestion
}
#advancedContextPlaceholders table {
	border-color: var(--color-base-80);
}
#advancedContextPlaceholders th,
```